nilearn/datasets/func.py

```diff
@@ -1275,3 +1279,229 @@ def fetch_mixed_gambles(n_subjects=1, data_dir=None, url=None, resume=True,
+            Matched image metadata will trigger actual image download.
+        If a dict, each key-value pair is an API filter.
+            The key will be checked for equality to the value.
+            Keys are applied before metadata download, limiting
+            the number of rows returned and possibly the number
+            of round trips made.
+        (see http://neurovault.org/api-docs#collapseImages for
+         API keys and collection metadata keys)
+
+    data_dir: string, optional (default None)
+        Path of the data directory. Used to force data storage in a specified
+        location. Default: None.
+
+    url: string, optional (default None)
+        Override download URL. Used for test only (or if you setup a mirror of
+        the data).
+
+    refresh: bool, optional (default False)
+        All metadata
+        If True, try requerying the database for new results.
+
+    resume: bool, optional (default True)
+        If True, try resuming download if possible.
+
+    verbose: int, optional (default 0)
+        Defines the level of verbosity of the output.
+
+    Returns
+    -------
+    data: Bunch
+        Dictionary-like object, the interest attributes are :
+        'collections': dict of dicts (one per collection)
+            Metadata about each collection (key: collection ID)
+            See http://neurovault.org/api-docs#collapseCollections
+            for all available fields.
+        'images': list of dicts
+            Metadata of image; parallel array to func_files.
+            See http://neurovault.org/api-docs#collapseImages
+            for available fields.
+            Also includes `local_path` (path to downloaded image)
+            and `collection_id` (which indexes into `collections`)
+        'func_files': list of strings
+            Paths to betamaps.
+
+    References
+    ----------
+    [1] Gorgolewski KJ, Varoquaux G, Rivera G, Schwartz Y, Ghosh SS, Maumet C,
+        Sochat VV, Nichols TE, Poldrack RA, Poline J-B, Yarkoni T and
+        Margulies DS (2015) NeuroVault.org: a web-based repository for
+        collecting and sharing unthresholded statistical maps of the human
+        brain. Front. Neuroinform. 9:8.
+        doi: 10.3389/fninf.2015.00008
+    """
+    import requests
+
+    if url is None:
+        url = "http://neurovault.org/api"
+    if collection_filters is None:
+        collection_filters = {}
+    if image_filters is None:
+        image_filters = {}
+    data_dir = _get_dataset_dir('neurovault',
+                                data_dir=data_dir)
+
+    def _build_nv_url(base_url, filts=None):
+        # Build a URL with the given filters.
+        if filts and isinstance(filts, dict):
+            url = '?'.join(base_url,
+                           '&'.join(['='.join(it) for it in filts.items()]))
+        else:
+            url = base_url
+        return url
+
+    def _get_nv_json(url, local_file=None, refresh=False):
```

Comment:
Yep, should definitely try to migrate this. If it requires tweaks to the fetching functions, will submit those as a separate PR.